### PR TITLE
Replace "here" link titles with meaningful titles [ci skip]

### PR DESCRIPTION
**Pull request by neanias**
### Motivation / Background

This Pull Request has been created because it's recommended not to use "here"/"click here"/"more" as the title for a link for accessibility reasons.[1]

[1]: https://www.gov.uk/guidance/content-design/links#writing-link-text

### Detail

This Pull Request changes link text in the migration guides.

### Additional information

When screen reader users are tabbing through all links on the page, they'll miss the context of "read more about X" preceding the "here". Thankfully, the sentences in the migration rewrite are already well-written so the "here" can be dropped in favour of the last part of the sentence to give the link title more meaning.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Unrelated changes should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

**Comment by neanias** [comment]
I don't think the docs-preview CI failure is due to my changes?

**Comment by neanias** [comment]
Also, I'm happy to sweep up other `[here]`s if it makes sense to do it in one big job?

**Comment by AmandaPerino** [comment]
> Also, I'm happy to sweep up other `[here]`s if it makes sense to do it in one big job?

Hi @neanias thanks for calling this out! The docs team agrees that this is a good idea and moving forward will be sure to create links more in line with this format.

If you're up for doing that sweep, we would love it if you could cast an eye over the guides that are already edited and see if there are other links that can be changed. The team will cover the as-yet-unedited guides as the documentation project progresses. 

Here is the list of guides that the team has already edited:

- [guides/source/active_record_callbacks.md](https://github.com/rails/rails/blob/main/guides/source/active_record_callbacks.md)
- [guides/source/action_view_overview.md](https://github.com/rails/rails/blob/main/guides/source/action_view_overview.md)
- [guides/source/action_view_helpers.md](https://github.com/rails/rails/blob/main/guides/source/action_view_helpers.md)
- [guides/source/active_record_basics.md](https://github.com/rails/rails/blob/main/guides/source/active_record_basics.md)
- [guides/source/active_model_basics.md](https://github.com/rails/rails/blob/main/guides/source/active_model_basics.md)
- [guides/source/action_text_overview.md](https://github.com/rails/rails/blob/main/guides/source/action_text_overview.md)
- [guides/source/action_mailbox_basics.md](https://github.com/rails/rails/blob/main/guides/source/action_mailbox_basics.md)
- [guides/source/active_job_basics.md](https://github.com/rails/rails/blob/main/guides/source/active_job_basics.md)

Thanks in advance if you can help!

File: guides/source/active_record_migrations.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ end
 We define the change that we want to happen moving forward in time. Before this
 migration is run, there will be no table. After it is run, the table will exist.
 Active Record knows how to reverse this migration as well; if we roll this
-migration back, it will remove the table. You can read more about rolling back
-migrations [here](#rolling-back).
+migration back, it will remove the table. You can read more about [rolling back
+migrations](#rolling-back).
```
Comment (p8) [inline guides/source/active_record_migrations.md:82]: The sentence reads a bit better if we remove the "You can", similar to the first change above.
```suggestion
migration back, it will remove the table. Read more about [rolling back
migrations](#rolling-back).
```
Or maybe:
```suggestion
migration back, it will remove the table. You can read more about rolling back migrations in
in the [Rolling Back](#rolling-back) section.
```

**Comment by neanias** [comment]
Can do!

File: guides/source/active_record_migrations.md (hunk 1 above)
Comment (neanias) [reply guides/source/active_record_migrations.md:82]: Thanks, I'll add with the other link changes!

**Comment by neanias** [comment]
Looks like the docs-preview CI task is stuck again. Is it worth cancelling the long running process?

**Comment by fatkodima** [comment]
Please, keep changes in 1 commit.

**Comment by neanias** [comment]
I'll squash them together now, one sec

File: guides/source/active_record_migrations.md [hunk 2]
```diff
@@ -832,8 +830,8 @@ In this example, we're updating the `price` column of the products table to
 WARNING: Modifying data directly in migrations should be approached with
 caution. Consider if this is the best approach for your use case, and be aware
 of potential drawbacks such as increased complexity and maintenance overhead,
-risks to data integrity and database portability. You can read more about data
-migrations [here](#data-migrations).
+risks to data integrity and database portability. You can read more about [data
+migrations](#data-migrations).
```
Comment (p8) [inline guides/source/active_record_migrations.md:834]: I like the phrasing of "You can read more about..." but this sentence ends abruptly now.
It's a bit wordy and doesn't flow as good, but maybe:
```suggestion
risks to data integrity and database portability. You can read the [data
migrations documentation](#data-migrations) for more details.
```

Or maybe just:
```suggestion
risks to data integrity and database portability. See the [data migrations
documentation](#data-migrations) for more details.
```

File: guides/source/active_record_migrations.md [hunk 3]
```diff
@@ -1256,8 +1254,8 @@ only perform the necessary tasks once.
   `bin/rails db:setup` does.
 * If the database exists but the tables have not been created, the command will
   load the schema, run any pending migrations, dump the updated schema, and
-  finally load the seed data. You can read more about seeding data
-  [here](#migrations-and-seed-data)
+  finally load the seed data. You can read more about [seeding
+  data](#migrations-and-seed-data).
```
Comment (p8) [inline guides/source/active_record_migrations.md:1258]: Similar to other suggestions...
```suggestion
  finally load the seed data. See the [seeding data
  documentation](#migrations-and-seed-data) for more details.
```

**Comment by p8** [comment]
@neanias Is "See X for more" also a bit problematic for visually impaired readers?

**Comment by neanias** [comment]
@p8, good question. My understanding is that it's better to link to titles or relevant terms when linking out. With the above example of "See the _seeding data documentation_ for more details", the link being on "seeding data documentation" makes sense out of context still so is a good candidate for link text.

File: guides/source/active_model_basics.md [hunk 4]
```diff
@@ -1520,8 +1521,9 @@ Objects you pass in are expected to return a compliant object from a call to
     end
     ```
 
-You can find the test methods
-[here](https://api.rubyonrails.org/classes/ActiveModel/Lint/Tests.html).
+See [the test methods
+documentation](https://api.rubyonrails.org/classes/ActiveModel/Lint/Tests.html)
+for more.
```
[the remark this answers is not in the document]
Comment (neanias) [reply guides/source/active_model_basics.md:1526]: I'd say "more details" is the most common usage in English

File: guides/source/action_text_overview.md [hunk 5]
```diff
@@ -27,8 +27,8 @@ Action Text includes a [WYSIWYG](https://en.wikipedia.org/wiki/WYSIWYG) editor
 called Trix, which is used in web applications to provide users with a
 user-friendly interface for creating and editing rich text content. It handles
 everything from providing enriching capabilities like the formatting of text,
-adding links or quotes, embedding images, and much much more. You can view an
-example of the Trix editor [here](https://trix-editor.org/).
+adding links or quotes, embedding images, and much much more. See [an
+example of the Trix editor](https://trix-editor.org/).
```
[the remark this answers is not in the document]
Comment (neanias) [reply guides/source/action_text_overview.md:31]: Yeah, I was struggling to think of a good way to make that sentence work. This feels better!

File: guides/source/active_record_migrations.md (hunk 2 above)
Comment (neanias) [reply guides/source/active_record_migrations.md:834]: I prefer the instructive "See..."

File: guides/source/action_view_overview.md [hunk 6]
```diff
@@ -846,5 +846,4 @@ directory. For example, setting `I18n.locale = :de` and creating
 `public/500.de.html` and `public/404.de.html` would allow you to have localized
 rescue pages.
 
-You can read more about the Rails Internationalization (I18n) API
-[here](i18n.html).
+You can read more about the [Rails Internationalization (I18n) API](i18n.html).
```
[the remark this answers is not in the document]
Comment (neanias) [reply guides/source/action_view_overview.md:849]: I'm going to move these "You can read..." to "See X".

File: guides/source/active_record_callbacks.md [hunk 7]
```diff
@@ -550,8 +550,8 @@ You have initialized an object!
 ### `after_touch`
 
 The [`after_touch`][] callback will be called whenever an Active Record object
-is touched. You can read more about `touch`
-[here](https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-touch).
+is touched. You can [read more about
+`touch`](https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-touch).
```
Comment (AmandaPerino) [inline guides/source/active_record_callbacks.md:554]:
```suggestion
`touch` in the API docs](https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-touch).
```

File: guides/source/active_record_callbacks.md [hunk 8]
```diff
@@ -178,8 +178,8 @@ order in which they will get called** during the respective operations:
 * [`after_save`][]
 * [`after_commit`][] / [`after_rollback`][]
 
-`after_commit` / `after_rollback` examples can be found
-[here](active_record_callbacks.html#after-commit-and-after-rollback).
+Find [examples for using `after_commit` /
```
Comment (AmandaPerino) [inline guides/source/active_record_callbacks.md:181]:
```suggestion
Jump to the [`after_commit` /
`after_rollback` section below](active_record_callbacks.html#after-commit-and-after-rollback) for examples using these two callbacks.
```

File: guides/source/active_record_callbacks.md [hunk 9]
```diff
@@ -363,8 +363,8 @@ create/update operations. It's important to note that save callbacks will always
 run before/around/after the more specific create/update callbacks.
 
 We've already covered [validation](#validation-callbacks) and
-[save](#save-callbacks) callbacks. `after_commit` / `after_rollback` examples
-can be found [here](#after-commit-and-after-rollback).
+[save](#save-callbacks) callbacks. Find [examples for using `after_commit`
```
Comment (AmandaPerino) [inline guides/source/active_record_callbacks.md:366]:
```suggestion
Jump to the [`after_commit` /
`after_rollback` section below](active_record_callbacks.html#after-commit-and-after-rollback) for examples using these two callbacks.
```

**Comment by AmandaPerino** [comment]
Thanks for this @neanias! I made a few suggestions for readability. 

@p8 I see what you mean about using "See here". We can also use "Please refer to", but in general I think people might be pretty understanding about this.

File: guides/source/active_record_callbacks.md (hunk 8 above)
Comment (neanias) [reply guides/source/active_record_callbacks.md:181]: I think I'm going to leave out the "below" (and next) if you're ok with that?

**Comment by neanias** [comment]
Slightly tweaked your suggestions @AmandaPerino in favour of using "See ... section"

File: guides/source/active_record_migrations.md [hunk 10]
```diff
@@ -88,8 +88,8 @@ the table, the concept of reversibility becomes crucial. With reversible
 migrations, not only does the migration create the table when applied, but it
 also enables smooth rollback functionality. In case of reverting the migration
 above, Active Record intelligently handles the removal of the table, maintaining
-database consistency throughout the process. You can read more about reversing
-migrations [here](#using-reversible).
+database consistency throughout the process. See the [reversing
+migrations section](#using-reversible) for more details.
```
Comment (p8) [inline guides/source/active_record_migrations.md:92]: For consistency I think we should capitalize section names:
```suggestion
database consistency throughout the process. See the [Reversing
Migrations section](#using-reversible) for more details.
```

File: guides/source/active_record_migrations.md [hunk 11]
```diff
@@ -621,8 +620,8 @@ This changes the column `part_number` on products table to be a `:text` field.
 
 NOTE: The `change_column` command is **irreversible**. To ensure your migration
 can be safely reverted, you will need to provide your own `reversible`
-migration. You can read more about reversible migrations
-[here](#using-reversible).
+migration. See the [reversible migrations section](#using-reversible) for more
+details.
```
Comment (p8) [inline guides/source/active_record_migrations.md:624]: For consistency I think we should capitalize section names:
```suggestion
migration. See the [Reversible Migrations section](#using-reversible) for more
details.
```

File: guides/source/active_record_migrations.md [hunk 12]
```diff
@@ -832,8 +831,8 @@ In this example, we're updating the `price` column of the products table to
 WARNING: Modifying data directly in migrations should be approached with
 caution. Consider if this is the best approach for your use case, and be aware
 of potential drawbacks such as increased complexity and maintenance overhead,
-risks to data integrity and database portability. You can read more about data
-migrations [here](#data-migrations).
+risks to data integrity and database portability. See the [data migrations
+documentation](#data-migrations) for more details.
```
Comment (p8) [inline guides/source/active_record_migrations.md:835]: For consistency I think we should capitalize section names:
```suggestion
risks to data integrity and database portability. See the [Data Migrations
documentation](#data-migrations) for more details.
```

File: guides/source/active_record_migrations.md [hunk 13]
```diff
@@ -1256,8 +1255,8 @@ only perform the necessary tasks once.
   `bin/rails db:setup` does.
 * If the database exists but the tables have not been created, the command will
   load the schema, run any pending migrations, dump the updated schema, and
-  finally load the seed data. You can read more about seeding data
-  [here](#migrations-and-seed-data)
+  finally load the seed data. See the [seeding data
```
Comment (p8) [inline guides/source/active_record_migrations.md:1258]: For consistency I think we should capitalize section names:
```suggestion
  finally load the seed data. See the [Seeding Data
```

**Review by p8 — COMMENTED** [review]
Just some minor comments left.

**Comment by p8** [comment]
Thanks @neanias !

**Comment by neanias** [comment]
No problem! Excited to have contributed to Rails after all these years.
